src/mmw/js/src/modeling/views.js

```diff
@@ -199,6 +258,8 @@ var ToolbarTabContentsView = Marionette.CollectionView.extend({
     childView: ToolbarTabContentView,
 
     onRender: function() {
+        // TODO: Move this logic to the template (active tab should
+        // be stored in model state somewhere).
```

Comment:
Sorry, it got rebased away. Definitely sounds like "initial scenario" and "active scenario slug" could be combined. I'll think it through a little more.